lib/tara/installer.rb

```diff
@@ -45,6 +57,7 @@ def bundle_gems
       FileUtils.mkdir_p(vendor_path)
       copy_gem_files(vendor_path)
       Dir.chdir(@package_dir) do
+        copy_local_gems
```

Comment:
Wouldn't it be possible to move this line into the `Bundler.with_clean_env` block a line down and then remove the `Bundler.with_clean_env` lines in `#find_installed_gems` and `#copy_local_gems`?